tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
+
+benchmark() {
+  local flags=$@;
```

Comment:
Or just `"$LEGOFY" "$@"` if we don't have default flags as suggested by comment below.